From dafdd0fd2bf7ce1aa2806a02a40aa5b8350e9ca0 Mon Sep 17 00:00:00 2001
From: Murali Karicheri <m-karicheri2@ti.com>
Date: Fri, 21 Sep 2018 16:10:55 -0400
Subject: [PATCH] hsr/prp: remove the debug print from the code.

This debug print causes a parser error in iec62439 mib parser
that expects the iplink -d output to be in a certain format to
extract the hsr/prp interface details. This was a left over
code from debugging which is removed to fix the issue. This print
appears when iplink -d command is executed causing the issue.

Upstream-Status: Pending

Signed-off-by: Murali Karicheri <m-karicheri2@ti.com>
---
 ip/iplink_hsr.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ip/iplink_hsr.c b/ip/iplink_hsr.c
index 2153494..34e9167 100644
--- a/ip/iplink_hsr.c
+++ b/ip/iplink_hsr.c
@@ -134,7 +134,6 @@ static void hsr_print_opt(struct link_util *lu, FILE *f, struct rtattr *tb[])
 	if (!tb)
 		return;
 
-	printf("%s\n", __func__);
 	if (tb[IFLA_HSR_SLAVE1] &&
 	    RTA_PAYLOAD(tb[IFLA_HSR_SLAVE1]) < sizeof(__u32))
 		return;
-- 
2.7.4

